apps-devstg/us-east-1/k8s-kops/1-prerequisites/terraform.auto.tfvars

```diff
@@ -0,0 +1,3 @@
+ssh_pub_key_path="/home/jdelacamara/Dev/work/J_and_J/security/aws-instances-ssh-keys/aws_instances_binbash.pub"
```

Comment:
@juanmatias may be something like ` ~/.ssh/bb/id_rsa` would be a better placeholder, and adding an inline comment to give more context to the user could also help.

Reply:
same as previous comment, fixed, was my own value